cmd/integration_tests/eventloop/eventloop_test.go

```diff
@@ -23,7 +30,7 @@ func eventLoopTest(t *testing.T, script []byte, testHandle func(context.Context,
 	logHook := &testutils.SimpleLogrusHook{HookedLevels: []logrus.Level{logrus.InfoLevel, logrus.WarnLevel, logrus.ErrorLevel}}
 	logger.AddHook(logHook)
 
-	script = []byte(`import {setTimeout} from "k6/experimental";
+	script = []byte(`import {setTimeout} from "k6/x/events";
   ` + string(script))
 	registry := metrics.NewRegistry()
 	builtinMetrics := metrics.RegisterBuiltinMetrics(registry)
```

Comment:
This and `js.New()` is unnecessary and make this into not-really-an-integration test :sweat_smile: 

If you make `newGlobalTestState()` and `newRootCommand()` exported in `cmd/`, you can use them here and write full integration tests that are actually shorter to write, since they won't need all of this setup, won't need the `init()` code, etc. From what I can see, you are mostly looking at log lines, which is super easy to do with that.

Reply:
I will need to move `newGlobalTestState` in separate package so that we don't pollute the global scope with the  `testing` package.

Reply:
:thinking: hmm yeah, maybe to this root `tests/` package? get it completely out of `cmd/` :man_shrugging: